### DISCUSSION TITLE
MQTT set value COMMAND_CLASS_SOUND_SWITCH with options

### DISCUSSION THREAD

**Discussion by Myriades-ll** [zwave-js/zwave-js-ui#2833] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Hi.
It would be great if you could add 2 more subtopic in the mqtt broker. 
- `numberOfTones` for the [getToneCount](https://github.com/zwave-js/node-zwave-js/blob/master/docs/api/CCs/SoundSwitch.md#gettonecount)
- `availableTones/tone_id/< name | duration >` for the [getToneInfo](https://github.com/zwave-js/node-zwave-js/blob/master/docs/api/CCs/SoundSwitch.md#gettoneinfo)
as described in [SoundSwitch.md](https://github.com/zwave-js/node-zwave-js/blob/master/docs/api/CCs/SoundSwitch.md)

**Describe the solution you'd like**
- `node_id/121/numberOfTones` = payload_with(value = n)
- `node_id/121/availableTones/tone_id/name` = payload_with(value="name")
- `node_id/121/availableTones/tone_id/duration` = payload_with(value=n)

**Describe alternatives you've considered**
Actually, i use the api/sendCommand. The above solution will simplify heavily further developments. It will also reduce the broker usage.

**Additional context**
Below a screenshot as example
![zwave](https://user-images.githubusercontent.com/82543406/205436414-c47dd06d-2994-4e02-be89-fbff72547e96.png)

Regards, Laurent

**Comment by robertsLando** [comment, marked as answer]
You can already specify options in payload, just add options property.

It is documented: https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=set-with-options
